### ISSUE TITLE
kitchensink, make sidelist link scroll to element id

### ISSUE THREAD

**Issue by StudioSpindle** [dorel/dorel.github.io#114]
Depends on #19 . At the moment sufficiënt elements have been added, so this #114 can be started.
------------------------

On the kitchensink there is a drawer panel on the left containing a index which shows all the elements that divide the page into separate sections. 

**requirements**
- Use css3 animation
- Make it a reusable component (maybe a iron-element transition of some sort)

**Expected outcome:**
When you click on a link, the page should scroll towards the area on the page. The transition should be fast but subtle. Also, it should feel organic, fast, slow, fast. 

Similar to: http://stackoverflow.com/a/35305697/2489480

Filenames are used for the id's of the `<section>` elements. So this same loop can be used for this animation. So perhaps use databinding for this: https://www.polymer-project.org/1.0/docs/devguide/data-binding

**Comment by StudioSpindle** [comment]
Won't fix
